[feat] 투표 생성 기능 추가

Nine-JH opened the pull request:
motivation:
- 요구사항 중 하나인 `스페이스의 회원은 투표를 생성할 수 있다`를 수행하기 위해 기능을 추가합니다.

## 변경사항
- `VoteRepository.java` 추가
- 투표 생성 API 추가

### Issue Link - #64

Nine-JH committed:
[feat] 투표 생성 기능 추가
motivation:
- 요구사항 중 하나인 `스페이스의 회원은 투표를 생성할 수 있다`를 수행하기 위해 기능을 추가합니다.

modification:
- `VoteRepository.java` 추가
- 투표 생성 API 추가

Nine-JH committed:
[feat] 투표 생성 기능 추가
motivation:
- 요구사항 중 하나인 `스페이스의 회원은 투표를 생성할 수 있다`를 수행하기 위해 기능을 추가합니다.

modification:
- `VoteRepository.java` 추가
- 투표 생성 API 추가

laigasus commented on app/src/main/java/fc/be/app/domain/vote/controller/dto/VoteCreateApiRequest.java:
```diff
@@ -0,0 +1,10 @@
+import jakarta.validation.constraints.Size;
+
+public record VoteCreateApiRequest(
+        @NotNull Long spaceId,
```
`@Positive` 도 추가하면 좋을 것 같아요!

Nine-JH replied:
확실히 더 단단한 코드가 되겠네요 감사합니다 😄

laigasus reviewed (approved):
LGTM 👍

Dr-KoKo reviewed (approved):
수고하셨습니다

happymink reviewed (approved):
고생하셨습니다!!

Nine-JH committed:
[feat] 투표 생성 기능 추가
motivation:
- spaceId는 항상 0보다 큰 값이기 때문에 @Positive 추가합니다.

modification:
- @Positive 추가

NoJaeHyuk reviewed (approved):
고생하셨습니다~
